docfx/doxygen_pages.cc

```diff
@@ -17,7 +17,6 @@
 #include <iostream>
```

Comment:
Late breaking news: this looks unnecessary here (and in doxygen2markdown.cc).

Reply:
Fixed in both places, and found a missing `#include` too.